videoplayer/static/html/videoplayer_view.html

```diff
@@ -5,11 +5,16 @@
         }
     </style>
     <h2>{{ display_name }}</h2>
-    <video data-dashjs-player src="{{ url }}" controls disablePictureInPicture controlsList="nodownload" width="100%" height="100%"
-        html5_sources="[&quot;{{ url }}&quot;]">
-        <p class="vjs-no-js">To view this video please enable JavaScript, and consider upgrading to a web browser that
-            <a href="http://videojs.com/html5-video-support/" target="_blank">supports HTML5 video</a></p>
-    </video>
+
+    <video
+        id="video"
```

Comment:
Is there any other unique identifier you can add to this id?  Like block id etc... If a single page has multiple xblocks then identifying these videos will be easy.